internal/pkg/block/block.go

```diff
@@ -31,10 +31,7 @@ type Block struct {
 	Ticket Ticket `json:"ticket"`
 
 	// ElectionProof is the vrf proof giving this block's miner authoring rights
-	ElectionProof crypto.VRFPi
-
-	// EPoStInfo wraps all data for verifying this block's Election PoSt
-	EPoStInfo EPoStInfo `json:"ePoStInfo"`
```

Comment:
This LGTM but don't we need a winning PoSts field to fully implement the protocol?  No need to add here as I'm working on this in another PR, but curious how we can interop with lotus blocks without post proofs.